src/test/dbwrapper_tests.cpp

```diff
@@ -42,6 +42,95 @@ BOOST_AUTO_TEST_CASE(dbwrapper)
     }
 }
 
+BOOST_AUTO_TEST_CASE(dbwrapper_basic_data)
+{
+    // Perform tests both obfuscated and non-obfuscated.
+    for (bool obfuscate : {false, true}) {
+        fs::path ph = SetDataDir(std::string("dbwrapper_1").append(obfuscate ? "_true" : "_false"));
+        //CDBWrapper dbw(ph, (1 << 20), true, false, obfuscate);
+        CDBWrapper dbw(ph, (1 << 20), false, true, obfuscate);
```

Comment:
nit: Could you comment these bool literal args?

Reply:
CDBWrapper dbw(ph, (1 << 20), false, true, obfuscate);